inst/rmarkdown/templates/oup/skeleton/skeleton.Rmd

```diff
@@ -23,69 +23,75 @@ author:
         - 2
 address:
   - code: ABC
-    address: Address ABC
+    department: Department of Biology
+    organization: University of Idaho
+    street: 123 Main Street
+    #postcode: 
+    city: Boise
+    state: Idaho 07458 # Mix and match elements to get proper address
+    country: United States
   - code: DEF
-    address: Address DEF
+    address: Address in one element deprecated, but works
   - code: Another University
     address: Address for Another University
 footnote:
   - code: 1
     text: "Equal contribution"
   - code: 2
-    text: "Current email address: \\href{mailto:cat@example.com}{cat@example.com}"
-abstract: |
-  This is the abstract.
-
-  It consists of two paragraphs.
+    text: "Current email address: [cat@example.com](mailto:cat@example.com)"
+abstract:
+  - "This is the first paragraph of the abstract. It has several sentences making it go over several lines. For this it needs to have a lot of text." 
+  - "This is the second paragraph."
+  - "**Motivation:** You can also have some paragraphs start with bold face."
+keywords:
+  - keyword1
+  - keyword2
+  - keyword3
+#boxedlist: # Not fully compatible with all document formats
+#  - First point
+#  - Second point
+competing_interests: |
+  There are no competing interest.
+author_contributions: |
+  AA did all the work. The others are just freeloaders.
 acknowledgements: |
   This is an acknowledgement.
 
   It consists of two paragraphs.
-keywords:
-  - key
-  - dictionary
-  - word
-#fontsize: 12pt
-#spacing: halfline # could also be oneline
 #classoptions:
-#  - endnotes
-bibliography: mybibfile.bib
+#  - namedate # Could also be here...
+bibliography: mybibfile.bib # Includes refs in OUP example template
 #csl: mycitationstyle.csl
-#link-citations: yes
-output: rticles::oup_article
+output:
+  rticles::oup_article:
+    keep_tex: TRUE
+    journal: "Journal Name"
+    #document_style: "contemporary" # Can be contemporary, modern, traditional; default "contemporary"
+    #papersize: "large" # Can be large, medium, small; default large
+    #citation_package: "default" # Uncomment if using a CSL biblio style; default "natbib"
+    namedate: TRUE # Set FALSE to use numeric refs; Default FALSE
+    #number_sections: FALSE # Uncomment to not number sections; default TRUE
+    #onecolumn: TRUE # Uncomment for one column format; default FALSE
```

Comment:
All this execept `number_sections` could be set as first level key in YAML header. 
What advantage to put them in the function format ?

Reply:
It is very format specific. I want to make it as effortless as possible to change from one journal format to another.

Reply:
I understand. we need to discuss that. We have the other thread.